package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "canvas",
   "description": "Canvas graphics API backed by Cairo",
-  "version": "1.5.0",
+  "version": "1.5.1",
```

Comment:
Please don't bump the version number, I'll do that as part of the release process